docs/demos/tutorials/03_Blocking.ipynb

```diff
@@ -794,6 +794,17 @@
    "source": [
     "linker.profile_columns(\"city || left(first_name,1) \")"
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "!!! note \"Further Reading\"\n",
+    "    :material-tools: For more on the blocking tools in Splink, please refer to the [Blocking API documentation](../../linkerbloc.md).\n",
```

Comment:
Is the link to `linkerbloc.md` still correct?

Reply:
Ignore this, I see you've moved it!

https://github.com/moj-analytical-services/splink/blob/0ca7b5a6c3b07a2b96440544dfc5753074f666bb/docs/linkerbloc.md